README.md

```diff
@@ -1,12 +1,19 @@
 # Questioner-ADC
-Crowd-source questions for a meetup. Set of API endpoints
```

Comment:
make sure to add the readme file in the gitignore.

Reply:
Okay will do so